packages/relay/tests/lib/eth.spec.ts

```diff
@@ -258,4 +352,40 @@ describe('Eth', async function () {
       expect(receipt).to.deep.eq(cachedReceipt);
     });
   });
+
+  describe('eth_getTransactionByHash', async function () {
+    it('returns `null` for non-existing hash', async function () {
```

Comment:
nit: mirror Node won't actually return null. It'll be a valid response with a message.
See [API Spec for getContractResultByTransactionIdOrHash](https://previewnet.mirrornode.hedera.com/api/v1/docs/#/contracts/getContractResultByTransactionIdOrHash)